javascript/javascript1/week3/noNoNoNoYes.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Javascript file for Hack Your Future, Javascript1, homework Week3
+ * program will take and save notes.
+*/
+
+// NOnoN0nOYes (Note taking app)
+
+// Save a note
+const notes = [];
+
+function saveNote(content, id) {
+    if (duplicateCheck(id)) {
```

Comment:
Here is an optional challenge if you feel like it. Instead of implementing your own `duplicateCheck` function, can you use the built-in Javascript function `find()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
thanks. it is great to learn new things. changes are done.